enterprise/backend/src/metabase_enterprise/sso/settings.clj

```diff
@@ -111,7 +111,6 @@ on your IdP, this usually looks something like `http://www.example.com/141xkex60
   (deferred-tru "Alias for the key that {0} should use for signing SAML requests"
                 (setting/application-name-for-setting-descriptions appearance/application-name))
   :encryption :when-encryption-key-set
-  :default    "metabase"
```

Comment:
this seemed to be an unnecessary default that prevented this value from actually being deleted?